Detectors/FIT/T0/macros/run_digi_fit.C

```diff
@@ -0,0 +1,96 @@
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+#include <sstream>
+
+#include <TStopwatch.h>
+
+#include "FairLogger.h"
+#include "FairRunAna.h"
+#include "FairFileSource.h"
+#include "FairRuntimeDb.h"
+#include "FairParRootFileIo.h"
+#include "FairSystemInfo.h"
+
+#include "FITSimulation/DigitizerTask.h"
+#endif
+
+void run_digi_fit(Int_t nEvents = 10, Float_t rate = 50.e3)
+{
+  // if rate>0 then continuous simulation for this rate will be performed
+
+  // Initialize logger
+  FairLogger* logger = FairLogger::GetLogger();
+  logger->SetLogVerbosityLevel("LOW");
+  logger->SetLogScreenLevel("DEBUG");
+
+  // Input and output file name
+  std::stringstream inputfile, outputfile, paramfile;
+  inputfile << "o2sim.root";
+  paramfile << "o2sim_par.root";
+  outputfile << "o2sim_digi.root";
+
+  // Setup timer
+  TStopwatch timer;
+
+  // Setup FairRoot analysis manager
+  FairRunAna* fRun = new FairRunAna();
+  FairFileSource* fFileSource = new FairFileSource(inputfile.str().c_str());
```

Comment:
I  would remove the macros relying on the FairRunAna chain. The DPL digitizer-workflow  should be used.